packages/stats-worker/src/YieldStats.ts

```diff
@@ -271,6 +272,24 @@ class YieldStats {
     return yieldData
   }
 
+  async addCachedDataToYieldData (yieldData: YieldData): Promise<YieldData> {
+    const url = `https://assets.hop.exchange/v1.1-pool-stats.json`
+    const cachedRes: any = await fetch(url)
+    const cachedData = await cachedRes.json()
+    for (const token in yieldData.pools) {
+      const tokenData = yieldData.pools[token]
+      for (const chain in tokenData) {
+        const chainData = tokenData[chain] 
+        // If APR is missing then the entire entry is missing
+        if (!chainData.apr) {
+          console.log(`Missing APR for ${chain}.${token}`, yieldData.pools[token][chain])
+          yieldData.pools[token][chain] = cachedData.data.pools[token][chain]
```

Comment:
suggestion: check that `cachedData.data.pools[token][chain]` exists before assigning

Reply:
Resolved with 85e91df74b5fdc4a94fcbb0ae85e6f9e26db0e18